js/demo.js

```diff
@@ -6,7 +6,8 @@ app.run(function($rootScope) {
   $rootScope.geobase = {
     'zoom': hash_loc.zoom,
     'lat' : hash_loc.center.lat,
-    'lon' : hash_loc.center.lng
+    'lon' : hash_loc.center.lng,
+    'locationProvided': hash_params
```

Comment:
```hash_params``` is accessible globally because ```L.Hash.parseHash(location.hash)``` is available and safe to access from ```$rootScope``` or ```$scope``` so, there is no reason to add ```locationProvided``` to ```$rootScope.geobase```